config/views.view.content.yml

```diff
@@ -176,71 +176,6 @@ display:
           settings:
             link_to_entity: true
           plugin_id: field
```

Comment:
why this change is required?

Reply:
No, it was not required. I removed the changes of that file.